code/modules/assembly/flash.dm

```diff
@@ -293,9 +293,14 @@
 		var/mob/living/silicon/robot/R = M
 		log_combat(user, R, "flashed", src)
 		update_icon(1)
-		R.Paralyze(70)
-		R.flash_act(affect_silicon = 1, type = /atom/movable/screen/fullscreen/flash/static)
-		user.visible_message("<span class='disarm'>[user] overloads [R]'s sensors with the flash!</span>", "<span class='danger'>You overload [R]'s sensors with the flash!</span>")
+		if(R.last_flashed + 30 SECONDS < world.time)
```

Comment:
30 seconds is huge, I'd say something like 10 or 5 seconds would be better. It should be long enough that the cyborg isnt instantly screwed again but quick enough that they won't enter another random fight in between

Reply:
We can give 10 a try but I really don't think that period of time is enough, especially if I also fulfill your second request of making it still blind them while they are stun resistant.

Reply:
Could try 15, in terms of our combat 15 seconds is a relatively long time but still a difficult amount of time to get into a situation where you are in a combat situation where the person attacking you doesnt know about the previous flashes. That, or there could be a special icon state showing when a flash doesnt work on the borg

Reply:
Look at testing evidence: borgs flash red when they have been recently flashed. This continues for the entire immunity period.

![dreamseeker_1djS2suuXp](https://user-images.githubusercontent.com/9547572/224563706-aad9c854-50e1-44b7-971a-f1cddc4cf2af.gif)

I can still shorten the time if you think 30 is too long for balance reasons, but being confused about whether a Borg can be stunned or not should be very intuitive with minimal exposure.